src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -837,28 +970,26 @@ def main():
         print(f"Originally planned to upload {count} object files")
         print(f"Detected {count - wtus_len} object files already exist in workspace")
 
-        data_dir = _prepare_skd_job_dir_to_upload(conf, wait_to_upload_objs)
+        _prepare_skd_job_dir_to_upload(conf, wait_to_upload_objs)
         print(
-            f"{wtus_len} objects in {data_dir} are ready to upload to workspace {workspace_id}"
+            f"{wtus_len} objects in {conf.job_data_dir} are ready to upload to workspace {workspace_id}"
         )
 
         start = time.time()
-        if create_assembly_only:
-            uploaded_count = _upload_assembly_files_in_parallel(
-                AssemblyUtil(
-                    conf.callback_url, service_ver=au_service_ver, token=conf.token
-                ),
-                ws,
-                env,
-                workspace_id,
-                load_id,
-                collections_source_dir,
-                wait_to_upload_objs,
-                batch_size,
-                source_dir,
-            )
-        else:
-            raise NotImplementedError("Genome object uploader is not implemented yet")
+
+        uploaded_count = _upload_objects_in_parallel(
+            ws,
+            env,
+            workspace_id,
+            load_id,
+            collections_source_dir,
+            wait_to_upload_objs,
+            batch_size,
+            source_dir,
+            conf,
+            au_service_ver,   # TODO - add GFU service ver
+            upload_assembly_only=create_assembly_only
```

Comment:
I thought we weren't going to support uploading assemblies only any more? I don't have a huge objection, but if it's something we're not going to use and it makes the code more complicated we should drop it

Reply:
okay. I will drop everything regarding Assembly only case in the next PR.

Reply:
:+1: